docs/howto/how_to_use_ci_result.md

```diff
@@ -0,0 +1,40 @@
+# How to Use CI Result in Apollo
+Apollo PR can be merged depend on CI result and CLA.
+
+## What works will CI check?
+Apollo CI will run following steps:
+    1: build your PR code merged master
+    2: lint your PR code include cc, h, py, BUILD, etc
+    3: run all unit tests in apollo project
```

Comment:
Leverage some markdown viewer to check if it shows as expected. For example:
https://github.com/ApolloAuto/apollo/blob/3ef830a7aa70c5306831206bc4a6d6b828f73c80/docs/howto/how_to_use_ci_result.md

Prefer:
```
1. Build your PR code merged master
1. Lint your PR code include cc, h, py, BUILD, etc
1. Run all unit tests in apollo project
```